src/styles/_md-styles.scss

```diff
@@ -4,27 +4,24 @@
     align-content: baseline;
   }
 }
-
 .keyboard-key {
   padding: 3px 6px;
   background: #eee;
   border-radius: 3px;
   outline: 1px solid #ddd;
   color: black;
 }
-
-example, 
```

Comment:
@MyLightIsOn  this should be `example` as well. I only have been using <example> in most md files